bot/exts/moderation/stream.py

```diff
@@ -47,8 +48,9 @@ async def _reload_tasks_from_redis(self) -> None:
         """Reload outstanding tasks from redis on startup, delete the task if the member has since left the server."""
         await self.bot.wait_until_guild_available()
         items = await self.task_cache.items()
+        guild = self.bot.get_guild(Guild.id)
         for key, value in items:
-            member = self.bot.get_guild(Guild.id).get_member(key)
+            member = await members.get_or_fetch(guild, key)
 
             if not member:
```

Comment:
This section (in the `if not member:`, github wont let me comment on it directly :/) should probably be updated as fetch_member would already have been tried.

Reply:
Ah yea, good catch. Force pushed :)